requirements-base.txt

```diff
@@ -4,7 +4,7 @@ pymongo==3.7.2
 gevent>=1.2.2
 ujson>=1.33
 hiredis>=0.1.5
-psutil>=5.1.2,<=5.6.3  # https://github.com/giampaolo/psutil/issues/1659#issuecomment-586032229
```

Comment:
@ghabrielv looks like this was limited because a specific bug as per the [link](https://github.com/giampaolo/psutil/issues/1659#issuecomment-586032229) there.

Also if the issue is no longer a problem, isn't better to use `psutil>=5.1.2` to allow us to use the newer library?